winthrop/books/models.py

```diff
@@ -287,7 +296,13 @@ def index_data(self):
             'authors': [str(author) for author in self.authors()],
             # first author only, for sorting
             # FIXME: sort on last name first? not an ordered relationship currently
-            'author_exact': str(self.authors().first()) if self.authors().exists() else None,
+            # NOTE: author_exact is a copy field.
+            'author_sort': str(self.authors().first()) if self.authors().exists() else None,
+            'editor_exact': [str(editor) for editor in self.editors()],
```

Comment:
@bwhicks I can see why you did this, but it feels a bit weird to me. How do you feel about switching it to generic fields (editor, translator, subject) that we then copy into the exact fields and the main text search field?

Reply:
@rlskoeser That should be a lot cleaner. I'll make the change.